docs/organise/before/visas.md

```diff
@@ -0,0 +1,34 @@
+__Note that this information does not constitute official advice, merely outlines experiences previous hackathon organisers have gone through when trying to handle LOIs. You should always reach out to your university or students' union for advice regarding visas. The gov.uk website also has some useful advice.__
```

Comment:
Might be worth add the link for the page on the `gov.uk` website. I had a look but I don't really know what I'm looking for.

Reply:
Can't find anything on gov.uk about writing LOIs, but there is a lot on the visa process (which I included on the attend page).

Most of the guides I found where about inviting friends/family.